README.md

```diff
@@ -1,155 +1,134 @@
+##### Consensus
 
->Note: `mobilecoind` provides the wallet bindings in gRPC, so that you can write a wallet client in any language. We provide both a [python and java example](./mobilecoind/clients/).
+New transactions must be checked for attempts to counterfeit value before new *key images* and *utxos* can be added to the MobileCoin blockchain. Transactions are prepared by the user on their local computer or mobile device, and submitted to a secure enclave running on a *validator node* of their choice. The *validator node* checks the transaction and, assuming it believes the transaction is valid, shares it with other nodes in the MobileCoin network. The transaction is passed only to peer secure enclaves that can establish via remote attestation that they are running unmodified MobileCoin software on an authentic Intel processor. Each secure enclave replicates a state machine that adds valid transactions to the ledger in a deterministic order using a consensus algorithm called the MobileCoin Consensus Protocol.
 
-##### The MobileCoin Daemon
+The MobileCoin Consensus Protocol is a high-performance solution to the byzantine agreement problem that allows new payments to be rapidly confirmed. The `consensus-service` target binary uses Intel Software Guard eXtensions (Intel SGX) to provide defense-in-depth improvements to privacy and trust.
 
-`mobilecoind` syncs transactions from the ledger to your local machine.
+To learn how MobileCoin uses Intel SGX to provide integrity in Byzantine Fault Tolerant (BFT) consensus as well as forward secrecy to secure your privacy, see the [consensus/enclave](./consensus/enclave) crate. To build and run consensus, see the [consensus/service](./consensus/service) crate.
 
->Note: This is necessary in order to run a wallet.
+*Full validator nodes* additionally use the `ledger-distribution` target binary to publish a copy of their computed blockchain to content delivery networks (currently to Amazon S3 only). The public blockchain is a zero-knowledge data structure that consists only of *utxos*, *key images* and block metadata used to ensure consistency and to construct Merkle proofs. To build and run ledger distribution, see the [ledger/distribution](./ledger/distribution) crate.
 
-To build and run the MobileCoin Daemon, see the [mobilecoind](./mobilecoind/README.md) crate.
+*Watcher nodes* perform an essential role in the MobileCoin network by verifying the signatures that the *full validator nodes* attach to each block. In this way the *watcher nodes* continuously monitor the integrity of the decentralized MobileCoin network. A *watcher node* also maintains a complete local copy of the blockchain and provide an API for wallet or exchange clients.
 
-### FAQ
+To run a *watcher node*, build and run the [`mobilecoind`](./mobilecoind) daemon.
 
-1. Are my transactions still private in the event of an SGX compromise?
+## FAQ
```

Comment:
What do you think about moving this out to an `FAQ.md`? to make the readme more slim

Reply:
We discussed this idea previously and decided to keep the FAQ in the main README.